test/unit/specs/components/staking/PageValidator.spec.js

```diff
@@ -1,15 +1,29 @@
-import DelegationModal from "staking/DelegationModal"
-import UndelegationModal from "staking/UndelegationModal"
-import TmModal from "common/TmModal"
-import setup from "../../../helpers/vuex-setup"
+import {shallowMount, createLocalVue} from "@vue/test-utils"
 import PageValidator from "renderer/components/staking/PageValidator"
-import lcdClientMock from "renderer/connectors/lcdClientMock.js"
+import BigNumber from "bignumber.js";
 
-const { stakingParameters } = lcdClientMock.state
-
-// TODO refactor comming up, not doing too much here
+const stakingParameters = {
```

Comment:
I would move all the objects that references to payloads outside the spec file, so if the payload changes we have to update only in 1 place + all tests will fail in different suites, instead of just in one, every tests that need them imports it

Reply:
I agree, and they should be tested against the swagger spec probably. but can we do this in a follow up (based on your first changes).